models/users.py

```diff
@@ -26,6 +27,7 @@ class User(BaseModel):
     first_name: str
     last_name: str
     email: EmailStr
+    password: bytes
```

Comment:
While this may work, it will force us to work with raw `bytes` which can be kind of annoying especially when dealing with database stuff. Changing this to `str` will allow us to abstract this away.